common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -269,6 +269,13 @@ div.problem {
       }
     }
 
+    // Hides equation previews in symbolic response problems when printing
```

Comment:
One feature of symbolic inputs is a preview for students, as shown below. This change hides the preview from printing.
![screen shot 2015-04-03 at 11 21 58 pm](https://cloud.githubusercontent.com/assets/6232546/6991274/505436a2-da58-11e4-8ae1-360621fec93e.png)